ssbarnea/vscode-yaml
# fix/gh template

- modification_api
- Added modification api
- Added newline to commands
- Turn schema contributor into schema extension api
- Removed unneeded interfaces from extension.ts
- Fixed messed up modifications_actions import
- Update to 0.7.0
- Allow multiple matches for one yaml file with requestCustomSchema
- Update to 0.7.1
- Updated to 0.7.2
- Added a "yamlValidation" example in documentation
- Update README.md
- Update README.md
- Update to 0.8.0
- remove references to kedge
- Update to 0.9.0
- Bump diff from 3.4.0 to 3.5.0
- Update vscode-yaml testing to use the new vscode-test package
- #267 add label for scheme contributions
- fix import
- Fix travis to use newer version of node and typescript
- Add in skipLibCheck to tsconfig
- Check Jenkins
- Release 0.9.1 with fix for release process
- Implement tests for register contributor label arg (#319)
- Fix error in yaml.customTags documentation in README.md (#321)
- Catch errors that can be thrown by registered schema providers #323
- Apply suggestions from code review - typo and indentation
- Use an output channel to log error message #326
- Reuse the output channel of the Language Client #326
- Deploy to open-vsx.org during releases
- Revert "Catch errors that can be thrown by registered schema providers #323"
- Re-add multiple contributors error test
- fix npm dependency vulnerability
- Add eslint and prettier (#342)
- Add GitHub Action to build PR (#345)
- Fix all eslint warnings
- Release '0.10.0' (#350)
- Use nodejs '12.18.3' in jenkins job
- Update to 0.10.1
- Add 'editor.tabSize' to preference syncronization
- Release '0.11.0' (#363)
- Use latest yaml-ls
- Release '0.11.1'
- Add issue templates
- Add pull request template
- Change the folder structure of ISSUE_TEMPLATE and PULL_REQUEST_TEMPLATE
- Add in codeowners file
- Fix PR template location so that it is picked up
- Use 'next' version of yaml-ls (#378)
- Add '[yaml]' to settings syncronization
- Release '0.12.0' (#385)
- Allow array for fileMatch in yamlValidation contribution
- Update package-lock.json
- Release '0.13.0'
- Add in non-standard lsp request for resolving schemas on the client-side
- Set 'nodejs-12.20.0' for Jenkins
- Update the name of the content request notification to be: yaml/registerVSCodeContentRequest
- Check that 'publishToMarketPlace' is set diring Jenkins CI build
- Remove .travis.yml since its not needed anymore
- Update the path of vscode-yaml staging to be /snapshots/vscode-yaml. Publish to /stable/vscode-yaml/ when a release is made.
- Release '0.14.0' (#414)
- #144 Add '`' to surroundingPairs and autoClosingPairs
- #359 set output chanel to never reveal
- Release '0.15.0'
- Add custom text document content provider (#418)
- Fix runtime warning on configuration loading
- Release '0.16.0'
- Update ts-eslint
- Implement a way to determine if an extension is conflicting with VSCode-YAML
- Any particular extension should only be available in one information window
- Add in additional conflicting ids
- #445 set 'editor.autoIndent' to "keep" value
- Load schema content contributed by other extensions (#451)
- Release '0.17.0'
- #462 fix returning undefined from cache
- Update src/json-schema-content-provider.ts
- fix build
- Bump y18n from 4.0.0 to 4.0.1
- Update src/json-schema-content-provider.ts
- fix build
- Update Readme for consistency
- update vscode version
- Release '0.18.0'
- Fix GH Action badge
- Update README with way to disable CodeLens (#489)
- Require minimal npm version
- switch to using yarn
- Add support for maxItemsComputed setting. Fixes #485
- Fix launching watch task
- feat: add config to disable additional properties (#499)
- Release '0.19.0' (#506)
- add yarn install on jenkins
- feat: add setting for schema store url
- docs: update README
- Add telemetry (#490)
- Clean schema cache if file was deleted (#511)
- Release '0.19.1'
- Remove telemetry (#516)
- Release 0.19.2
- Use yarn on GH action, upload vsix artifact
- chore: remove executable flags
- remove deprecated 'configurationSection' (#522)
- Bump version
- Restore telemetry without the vscode-commons dependency
- #520 add GH workflow for default branch migration
- #520 use 'main' as default brunch in CI
- Release '0.20.0'
- Bump version to 0.21.0
- Update vscode-redhat-telemetry to 0.2.0
- feat: support workspace trust
- Improve telemetry (#543)
- Release 0.21.0
- bump version
- Release 0.21.1 (#555)
- Add capability to support virtual workspaces
- Release 0.22.0
- Bump version
- Remove joshuawilson from code owners
- Bump axios from 0.21.1 to 0.21.4
- Remove default branch migration workflow
- Update vscode-redhat-telemetry to 0.4.2
- Add code bundling with webpack (#596)
- Throttle LSP error telemetry reporting (#599)
- Release '0.23.0'
- Bump version
- Add setting for yaml spec version (#606)
- Release '1.0.0' (#607)
- Fix webpack bundling, by fixing prettify version
- add web support (#594)
- Bump 1.1.0 (#613)
- Fix running tests
- updated CONTRIBUTING.md
- Release '1.1.0'
- Bump version
- Fix completion test
- Change 'wordPattern' to be more yaml specific
- Release '1.1.1' (#635)
- Release '1.2.0'
- Bump version
- Upgrade 'request-light' version
- Start extension on 'dockercompose' language mode
- Release '1.2.1'
- Release 1.2.2
- Add status bar item for JSON schema selection (#643)
- Remove jpinkney from codeowners
- Fix json schema status bar item tests (#674)
- feat: add disableDefaultProperties (#667)
- integreated the latest 0.5.7 request-light lib version
- Bump follow-redirects from 1.14.4 to 1.14.7
- Release '1.3.0'
- Bump version
- Add 'telemetry' tag to redhat.telemetry.enabled preference
- Release 1.4.0
- Bump version 1.5.0
- Add msivasubramaniaan to codeowners
- Fix typos
- Bump follow-redirects from 1.14.7 to 1.14.8
- improve modeline doc
- Release 1.5.0
- Bump version
- release 1.5.1
- changed back to the version to 1.6.0
- update CHANGELOG.md
- Revert "update CHANGELOG.md"
- update CHANGELOG.md
- update bump version 1.7.0
- Bump minimist from 1.2.5 to 1.2.6
- Support JSON Schema versions from schemastore (#710)
- Alternate way to add local schema in windows
- Improvements to the contributing instructions
- fix indentation
- Send startup event on LS start (#744)
- Release 1.7.0
- Use sftp to upload vsix
- Fix upload to stable
- Bump version
- Bump async from 3.2.1 to 3.2.3
- Update issue creation config

### What does this PR do?


### What issues does this PR fix or reference?


### Is it tested? How?
